src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts

```diff
@@ -267,6 +265,18 @@ export class CommentTextareaComponent implements OnInit, AfterViewInit, OnChange
     });
   }
 
+  private clearPlaceholderIfNeeded(): void {
+    const currentText = this.commentTextarea.nativeElement.textContent.trim();
```

Comment:
nativeElement.textContent can return null or an unexpected value, which is not the ideal behavior:
```textContent?.trim() || ''````